docs/get-started.md

```diff
@@ -2,29 +2,39 @@
 
 ## Supported Platforms
 
-Sematic currently supports Linux and Mac. If you're using Windows, you can
+Sematic currently supports Linux and MacOS. If you're using Windows, you can
 run Sematic in
 [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
 
-Python versions 3.8, 3.9, and 3.10 are supported.
+Python versions `3.8`, `3.9`, and `3.10` are supported.
 
 ## Installation
 
-Install Sematic in your local Python enviroment with:
+Install Sematic in your local Python environment with:
 
 ```shell
 $ pip install sematic
 ```
 
-## Starting the web dashboard
+The above command will also install all required Python dependencies.
 
-Before running pipelines, you need to start the metadata server and web dashboard with:
+Sematic does require an additional system library installed, `libmagic` / `filemagic`,
+which is required by [`python-magic`](https://pypi.org/project/python-magic/):
+
+- MacOs: `brew install libmagic`, or `port install file`
```

Comment:
In my case, I already have this installed. But Bazel build still fails. Are there additional steps required? Eg. updating certain cache?

Reply:
Let's debug offline.